config/boards/radxa-e20c.csc

```diff
@@ -0,0 +1,13 @@
+# Rockchip RK3528 quad core 1-2GB SoC 2xGBe 8GB eMMC
+BOARD_NAME="Radxa E20C"
+BOARDFAMILY="rk35xx"
+BOOTCONFIG="radxa_e20c_rk3528_defconfig"
+BOARD_MAINTAINER="mattx433"
+KERNEL_TARGET="vendor"
+FULL_DESKTOP="no"
+BOOT_LOGO="desktop"
+BOOT_FDT_FILE="rockchip/rk3528-radxa-e20c.dtb"
+BOOT_SCENARIO="spl-blobs"
+WIREGUARD="no"
+IMAGE_PARTITION_TABLE="gpt"
+BOOTFS_TYPE="ext4"
```

Comment:
Is this boot partition necessary? The default size of boot will limit user switching different versions of kernel.

Reply:
It isn't - removed it in the next commit.